.github/workflows/tests.yaml

```diff
@@ -1,9 +1,9 @@
 name: unit tests
 
 on:
-  pull_request:
+  push:
     branches:
-      - main
+      - cwp/pg
```

Comment:
❓ You might want to revert this part of the PR

Reply:
Good point, thanks.